File: CodeChef/Self Learning/Basic Programming Concepts/Richie_Rich.cpp
// CodeChef - CHFRICH - Richie Rich
// https://www.codechef.com/LP1TO201/problems/CHFRICH

#include "bits/stdc++.h"
using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int T;
    cin >> T;
    int A, B, X;
    for(int i = 1; i <= T; ++i){
        cin >> A >> B >> X;
        int needed_growth = B - A;
        int ans = needed_growth / X;
        cout << ans << endl;
    }

}

File: CodeChef/Self Learning/Basic Programming Concepts/Richie_Rich.py
# CodeChef - CHFRICH - Richie Rich
# https://www.codechef.com/LP1TO201/problems/CHFRICH

import sys, math
input = sys.stdin.readline

T = int(input())
for _ in range(T):
    A, B, X = map(int, input().split())
    ans = 0
    needed_growth = B - A
    ans = needed_growth // X
    print(ans)
